packages/react-native/Libraries/AppDelegate/RCTAppDelegate.h

```diff
@@ -163,6 +163,13 @@ NS_ASSUME_NONNULL_BEGIN
 /// Return the bundle URL for the main bundle.
 - (NSURL *__nullable)bundleURL;
 
+/**
+ * It creates the RCTRootViewFactory.
+ *
+ * @return: an instance of `RCTRootViewFactory`.
+ */
+- (RCTRootViewFactory *)createRCTRootViewFactory;
+
```

Comment:
I don't think we should expose it, as this is an implementation detail of RCTAppDelegate. Probably we need proper support for the SceneDelegate scenario

Reply:
I've posted a longer explanation in the related issue: https://github.com/facebook/react-native/issues/46184#issuecomment-2311992358